src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,123 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  min-height: 406px;
```

Comment:
why do you need this?

Reply:
<img width="979" alt="Screen Shot 2022-10-13 at 10 59 08" src="https://user-images.githubusercontent.com/101462543/195537208-ceb67602-143a-4f26-86e9-ec1ce0ca9bb9.png">
I had this comment on previous task

Reply:
@stanislav-topikha 
Do I need set fixed height?

Reply:
Deleted it